deep-security-agent/README.md

```diff
@@ -0,0 +1,112 @@
+A collection of playbooks to help deploy and manage the Trend Micro Deep Security Agent.
+
+## Requirements
+
+All of the tasks in this repository require a working Deep Security infrastructure. The key component is the Trend Micro Deep Security Manager. The agents (which these playbooks help you manage) do the heavy lifting but the manager gives the marching orders. 
+
+There are no specific technical requirements beyond a standard Ansible deployment.
+
+
+## Dependencies
+
+No dependency is required.
+
+
+## Usage
+
+A sample playbook:
+
+```yaml
+- hosts: all
+  roles:
+    - role: deep-security.deep-security-agent
+      action: deploy
+      dsa_download_hostname: app.deepsecurity.trendmicro.com
+      dsa_download_port: 443
+      dsa_activation_hostname: agents.deepsecurity.trendmicro.com
+      dsa_activation_port: 443
+      tenant_id: 111A111A-1A1A-11AA-AAA-11AA11111111
+      token: 111A111A-1A1A-11AA-AAA-11AA11111111
+      policy_id: 1
+      force_reactivation: false
+```
+
+A sample playbook in JSON format:
```

Comment:
Not so sure if we need two samples, as the user can use either one of them without any dependency.

Reply:
After reviewing top 10 starred repos on Ansible Galaxy, most of their examples use pure Yaml format. Will remove JSON format to avoid confusing.